lnwallet/reservation.go

```diff
@@ -158,6 +158,13 @@ func NewChannelReservation(capacity, fundingAmt btcutil.Amount,
 		ourBalance = pushMSat
 		theirBalance = capacityMSat - feeMSat - pushMSat
 		initiator = false
+
+		// If the responder doesn't have enough funds to actually pay
+		// the fees, then we'll bail our early.
+		if int64(theirBalance) < 0 {
+			return nil, fmt.Errorf("remote balance (%v) not enough to "+
```

Comment:
must be rebased to use new `ReservationError`

Reply:
Will fix.